jina/drivers/search.py

```diff
@@ -31,24 +31,26 @@ def __init__(
 
 
 class KVSearchDriver(BaseSearchDriver):
-    """Fill in the doc/chunk-level top-k results using the :class:`jina.executors.indexers.meta.BinaryPbIndexer`
+    """Fill in the results using the :class:`jina.executors.indexers.meta.BinaryPbIndexer`
```

Comment:
Probably we could all use first-person expressions(like 'fill') or third person(like 'fills')? Now some docstrings use first-person but some use third-person

Reply:
Let's stick to the first-person expression